examples/templates/docker/README.md

```diff
@@ -9,70 +9,16 @@ tags: [local, docker]
 To get started, run `coder templates init`. When prompted, select this template.
 Follow the on-screen instructions to proceed.
 
-## Adding/removing images
+## Editing the image
 
-After building and pushing an image to an image registry (e.g., DockerHub), edit
-the template to make the image available to users:
-
-```sh
-# Open the template
-vim main.tf
-```
-
-Modify your file to match the following:
-
-```diff
-variable "docker_image" {
-  description = "What Docker image would you like to use for your workspace?"
-  default     = "codercom/enterprise-base:ubuntu"
-  validation {
-    condition     = contains(["codercom/enterprise-base:ubuntu", "codercom/enterprise-node:ubuntu",
--                              "codercom/enterprise-intellij:ubuntu"], var.docker_image)
-+                              "codercom/enterprise-intellij:ubuntu", "codercom/enterprise-golang:ubuntu"], var.docker_image)
-    error_message = "Invalid Docker image!"
-  }
-}
-```
-
-Update the template:
-
-```sh
-coder template push docker
-```
-
-You can also remove images from the validation list. Workspaces using older template versions will continue using
-the removed image until you update the workspace to the latest version.
+Simply edit the `Dockerfile` and run `coder templates push` to update workspaces.
```

Comment:
please remove the word "simply" im very anti simply because simple to you might not be simple to me  (creates unwelcoming spaces for learning)